go/vt/vtgate/executor.go

```diff
@@ -414,16 +414,17 @@ func (e *Executor) handleBegin(ctx context.Context, safeSession *SafeSession, sq
 	err := e.txConn.Begin(ctx, safeSession)
 	logStats.ExecuteTime = time.Since(execStart)
 
-	queriesProcessed.Add([]string{"Begin", "", ""}, 1)
+	queriesProcessed.Add("Begin", 1)
```

Comment:
minor suggestion -- I would still use `updateQueryCounts` here... just pass a 0 for ShardQueries